.github/workflows/setup-homebrew.yml

```diff
@@ -71,4 +71,4 @@ jobs:
 
       - run: brew test-bot --only-setup
 
-      - run: brew test-bot --only-tap-syntax
+      - run: brew info hello
```

Comment:
Why did this change? Want to make sure we're still testing the `brew test-bot` case here?

Reply:
Tap syntax checks require a Homebrew/core clone (and are not the quickest).

We already test cleanup and setup parts of test-bot here, which should cover if `brew` is set up correctly. But we could add `--test-default-formula` or something.

Reply:
> But we could add --test-default-formula or something if you feel more is needed.

Yeh, that'd be nice I think 👍🏻 

> Tap syntax checks require a Homebrew/core clone, which doesn't seem worth adding to what is already a long step taking several minutes.

Agreed 👍🏻